Do not hardcode rpath dirs into binaries or modules that would cause
unexpected results with pkgfs.

--- hints/netbsd.sh.orig	2008-10-02 01:03:43.000000000 +0200
+++ hints/netbsd.sh	2008-10-02 01:04:38.000000000 +0200
@@ -178,8 +178,8 @@ EOCBU
 # Set sensible defaults for NetBSD: look for local software in
 # /usr/pkg (NetBSD Packages Collection) and in /usr/local.
 #
-loclibpth="/usr/pkg/lib /usr/local/lib"
-locincpth="/usr/pkg/include /usr/local/include"
+loclibpth=""
+locincpth=""
 case "$rpathflag" in
 '')
 	ldflags=
